GVRf/Framework/jni/engine/renderer/renderer.cpp

```diff
@@ -194,7 +194,7 @@ void Renderer::renderCamera(Scene* scene, Camera* camera, int framebufferId,
 
     bool renderShadow = cameraLight; // TODO reader from scene;
 
-    if ((framebufferId != 0) && !renderShadow) {
+    if (!renderShadow) {
```

Comment:
Does the gvr-shadows sample program exhibit the rendering problem with the shadow maps if I run it on a monoscopic view?

From: Frederico Simas Duarte [mailto:notifications@github.com]
Sent: Friday, March 4, 2016 10:05 AM
To: Samsung/GearVRf <GearVRf@noreply.github.com>
Cc: Nola Donato <nola.donato@samsung.com>
Subject: Re: [GearVRf] Fix crash in monoscopic view applications. The first problem was a ze… (#442)


In GVRf/Framework/jni/engine/renderer/renderer.cpp<https://github.com/Samsung/GearVRf/pull/442#discussion_r55064879>:

> @@ -194,7 +194,7 @@ void Renderer::renderCamera(Scene* scene, Camera* camera, int framebufferId,

>

>      bool renderShadow = cameraLight; // TODO reader from scene;

>

> -    if ((framebufferId != 0) && !renderShadow) {

> +    if (!renderShadow) {

There was an issue with the monoscopic renderer and the rendering with the shadow map itself when we first did this a while back, but, i guess it can be removed now, since the crash is a critical issue.
I'll work on a adaptation of the shadow map with this renderer.

—
Reply to this email directly or view it on GitHub<https://github.com/Samsung/GearVRf/pull/442/files#r55064879>.